unstructured_inference/inference/layout.py

```diff
@@ -261,40 +229,6 @@ def get_elements_with_detection_model(
         # NOTE(mrobinson) - We'll want make this model inference step some kind of
         # remote call in the future.
         inferred_layout: List[LayoutElement] = self.detection_model(self.image)
-        inferred_layout = UnstructuredObjectDetectionModel.deduplicate_detected_elements(
```

Comment:
This call to deduplicate_detected_elements is required to handle nested elements generated by detection models

Reply:
oops i deleted too much